docs/products/cassandra/concepts/cross-cluster-replication.md

```diff
@@ -1,9 +1,10 @@
 ---
 title: Cross-cluster replication with Aiven for Apache Cassandra®
+pro: true
```

Comment:
Already raised it on slack, but just for being extra sure! Pro Platform is now Aiven Entreprise